cloudify/plugins/workflows.py

```diff
@@ -213,14 +213,9 @@ def auto_heal_reinstall_node_subgraph(
             ctx,
             node_instance_ids=[ni.id for ni in sub_node_instances],
             name='check_status',
+            ignore_failure=True,
         )
-        try:
```

Comment:
this is tested by `test_builtin_workflows.py::TestHealOperation::test_failing_status_heal_operation`